datadog_checks_base/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # CHANGELOG - datadog_checks_base
 
+## 25.5.1 / 2022-08-08
```

Comment:
`25.5.1` does not exist yet, only `25.6.0` https://github.com/DataDog/integrations-core/blob/master/datadog_checks_base/CHANGELOG.md#2560--2022-07-11 👍